common/static/css/edx-cc.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'edx-cc';
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265');
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265#iefix') format('embedded-opentype'),
+       url('../fonts/edx-cc/edx-cc.woff?52318265') format('woff'),
+       url('../fonts/edx-cc/edx-cc.ttf?52318265') format('truetype'),
+       url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
+/*
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+  @font-face {
+    font-family: 'edx-cc';
+    src: url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  }
+}
+*/
+
+ [class^="icon-cc"]:before, [class*=" icon-cc"]:before {
+  font-family: "edx-cc";
+  font-style: normal;
+  font-weight: normal;
```

Comment:
NIT: not sure that these two font rules (style/ weight) are necessary. your call on if they're safe to remove.